tests/test_requests.py

```diff
@@ -1533,28 +1520,24 @@ def test_expires_invalid_int(self, value, exception):
 
     def test_expires_none(self):
         """Test case where expires is None."""
-
         morsel = Morsel()
         morsel['expires'] = None
         cookie = morsel_to_cookie(morsel)
         assert cookie.expires is None
 
 
 class TestMorselToCookieMaxAge:
-
     """Tests for morsel_to_cookie when morsel contains max-age."""
 
     def test_max_age_valid_int(self):
         """Test case where a valid max age in seconds is passed."""
-
         morsel = Morsel()
         morsel['max-age'] = 60
         cookie = morsel_to_cookie(morsel)
         assert isinstance(cookie.expires, int)
 
     def test_max_age_invalid_str(self):
         """Test case where a invalid max age is passed."""
-
```

Comment:
Alright, I'll drop c50eb62 for now and possibly make another pass at fixing any glaring space issues in a separate PR tied to the parent issue.